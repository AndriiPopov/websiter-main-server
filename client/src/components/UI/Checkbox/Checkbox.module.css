.Div {
    background: none;
    outline: none;
    border: none;
    font-size: 12px;
    display: block;
    height: initial;
    padding: 4px 5px;
}

.Input {
    width: 15px;
    height: 15px;
    margin-left: 10px; 
}

.StyleControls {
    display: inline-block;
}