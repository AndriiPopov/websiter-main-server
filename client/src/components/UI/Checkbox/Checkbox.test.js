import React from 'react';
import {mount, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import { Checkbox } from './Checkbox';

configure({adapter: new Adapter()});

describe('Checkbox', () => {
    it('should be defined', () => {
        expect(Checkbox).toBeDefined();
    });

    it('it should render button with onclick null when disabled', () => {
        const tree = mount(
            <Checkbox 
                checked
                sectionValues="{}"
                boxesValues="{}"
                changedStyleValues="[]" />
        );
        expect(toJson(tree)).toMatchSnapshot();
    });
});