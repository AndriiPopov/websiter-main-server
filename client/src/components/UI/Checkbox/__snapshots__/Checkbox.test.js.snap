// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox it should render button with onclick null when disabled 1`] = `
<Checkbox
  boxesValues="{}"
  changedStyleValues="[]"
  checked={true}
  sectionValues="{}"
>
  <div
    className="Div"
  >
    <input
      checked={true}
      className="Input"
      onChange={[Function]}
      type="checkbox"
    />
  </div>
</Checkbox>
`;
